.github/workflows/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Github action to create EKS cluster using eksctl and install DOP on it
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install eksctl
+      run: |
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+        sudo mv /tmp/eksctl /usr/local/bin
+        sleep 10
+        eksctl version
+
+    - name: Checkout code
+      uses: actions/checkout@master
+
+    - name: Run Shellcheck
+      uses: bewuethr/shellcheck-action@v2
+
+##     TODO:: There is no go-file in repo. Thatswhy we are
+##     commenting this lines for now. 
+#
+#     - name: Run golangci-lint
+#       uses: actions-contrib/golangci-lint@v1
+
+    - name: Configure AWS credentials from secrets
+      uses: aws-actions/configure-aws-credentials@v1
```

Comment:
What does this action do?
Can we write in comments

Reply:
I am hesitant to have so much of intelligence in github actions.
Can you install EKS without github actions?
can you install EKS from your laptop using our scripts?

Reply:
created a seperate file. Now it is completely independent with github actions @AmitKumarDas